qemu/tests/cfg/block_resize.cfg

```diff
@@ -10,9 +10,7 @@
     create_image_stg = yes
     force_create_image_stg = yes
     blk_extra_params_stg = "serial=TARGET_DISK0"
```

Comment:
The issue is, on power platform, the device kernel name of virtual hard drive 'stg' not always be '*db' (it is 'vda' for using virtio-blk interface). We need to get its kname dynamically, so I wrote commit f83f336. But, sorry, I forgot to put the param into all the related cases..